backup/syncer/job.go

```diff
@@ -152,13 +156,15 @@ func (s *jobSyncer) ensurePodSpec(in corev1.PodSpec) corev1.PodSpec {
 			"/bin/bash", "-c", "--",
 		}
 		backupToDir, DateTime := utils.BuildBackupName(s.backup.Spec.ClusterName)
+		// add the gtid script
 		strAnnonations := fmt.Sprintf(`curl -X PATCH -H "Authorization: Bearer $(cat /var/run/secrets/kubernetes.io/serviceaccount/token)" -H "Content-Type: application/json-patch+json" \
```

Comment:
Suggest a more native way to add annotations 😊

Reply:
it will do it next time.